openedx/core/djangoapps/appsembler/api/v1/views.py

```diff
@@ -171,14 +171,14 @@ def create(self, request):
 
         # set the honor_code and honor_code like checked,
         # so we can use the already defined methods for creating an user
-        data['honor_code'] = True
-        data['terms_of_service'] = True
+        data['honor_code'] = 'True'
+        data['terms_of_service'] = 'True'
 
         if password_provided:
             try:
                 # Default behavior is True - send the email
-                data['send_activation_email'] = strtobool(
-                    str(data.get('send_activation_email', True)))
+                data['send_activation_email'] = bool(strtobool(
```

Comment:
I think using `bool(strtobool(str()))` is an overkill and hard to really test well for something that should be done with plain if statements. It has broken once and I don't think it will be easy to modify in the future.

```suggestion
    def _normalize_send_activation_email(self, value):
        """
        Allow string `False`/`True` to be used by the API caller.
        """
        if value == "False":
            value == False
        elif value == "True":
            value = True

        if not (value == False or value == True):
            # Return 400 Bad Request to the API caller
            raise SomeKindOfException("Invalid request")

        return value

    def create(self, request):
        # ...
        data['send_activation_email'] == _normalize_send_activation_email(data['send_activation_email'])
        # ...

```

If you'd like, you can also document those changes with inline comments:

```
    # TODO: Backward compatibility for reported issue in RED-113 until both of the
    #       customers and documentations are up to date with that _breaking_ change
```